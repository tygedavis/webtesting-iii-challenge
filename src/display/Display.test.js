import React from 'react';
import { render } from '@testing-library/react';

import Display from './Display';

test('Displays if gate is open/closed', () => {
  const locked = true;
  const closed = true;
  const tree = render(<Display 
    locked={locked}
    closed={closed}
    />)
  expect(tree.baseElement).toMatchSnapshot();
})