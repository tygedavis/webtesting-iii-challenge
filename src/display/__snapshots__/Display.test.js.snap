// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays if gate is open/closed 1`] = `
<body>
  <div>
    <div
      class="display panel"
    >
      <div
        class="led red-led"
      >
        Locked
      </div>
      <div
        class="led red-led"
      >
        Closed
      </div>
    </div>
  </div>
</body>
`;
